mfr/core/utils.py

```diff
@@ -135,7 +134,11 @@ def get_exporter_name(name: str) -> str:
     ep_iterator = pkg_resources.iter_entry_points(group='mfr.exporters', name=name.lower())
     ep_list = list(ep_iterator)
 
-    # `make_renderer()` is called before `make_exporter()` to ensure the file type is supported
+    # Empty list indicates unsupported export type.  Return '' and let `make_exporter()` handle it.
+    if len(ep_list) == 0:
```

Comment:
The fix follows exactly the same code used in `get_renderer_name()`. In addition, the comment is corrected.